frontends/web/src/app.tsx

```diff
@@ -162,6 +162,10 @@ class App extends Component<Props, State> {
             route(`/device/${deviceIDs[0]}`, true);
             return;
         }
+        if (deviceIDs.length === 0 && currentURL.startsWith('/buy/')) {
```

Comment:
isn't `accounts.length === 0` better, like above with the account summary? the buy component relies on accounts. not devices.